modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-okhttp/infrastructure/ApiClient.kt.mustache

```diff
@@ -77,6 +74,10 @@ import com.fasterxml.jackson.core.type.TypeReference
 import com.squareup.moshi.adapter
 {{/moshi}}
 
+{{#jvm-okhttp4}}
+val EMPTY_REQUEST: RequestBody = ByteArray(0).toRequestBody()
```

Comment:
Hi @noordawod, thanks for creating this PR.
Can you please add the prefix `{{#nonPublicApi}}internal {{/nonPublicApi}}` and update the sample projects? 
Thanks

Reply:
Will do, thanks for the help.

Reply:
✅